### DISCUSSION TITLE
Issue at Build/KnowledgeMW

### DISCUSSION THREAD

**Discussion by gow1998** [Sunbird-coKreat/Community#66] [Bugs]
Hello team,
       I am facing an issue while build the KnowledgeMW service. I used release-5.2.0 branch as per mentioned in the cokreat documentation and also tried with branch release-5.3.0, still I have the issue. 
![image](https://user-images.githubusercontent.com/115560712/228866073-6c52d60c-042c-4957-953a-2645774179d3.png)
Please help me to resolve this issue

**Comment by Aparna-karri591** [comment]
@gow1998 kindly post it in the knowledge discussion forum

**Reply by gow1998** [reply]
@Aparna-karri591 ok sure
